### PR TITLE
Adding script start instead of geth

### PR DESCRIPTION
**Pull request by edisinovcic**
Changing start of the geth dockerfile from geth command to running script so we are more flexible.

### CODE REVIEW

**Comment by edisinovcic** [comment]
@chriseth should I revert it to the older version (just running geth with command as goerli is also supoorted via --goerli flag).

**Comment by chriseth** [comment]
To be honest, I don't know what you are trying to do here :)
Wouldn't it be more suited to put the dockerfile and the script somewhere else and only reference this branch from inside the dockerfile?

**Comment by edisinovcic** [comment]
Original dockerfiles start command is "geth".
 
I replaced "geth" from CMD to running a scripts so it's more universal way run any network.

Also script can be easily overriden by mounting a volume from host to docker container on a place where script is and that way replace and more easily run multiple different networks in different containers. Same goes for genesis.json file etc. 

@ligi mentioned that goerli is supported by geth via command --goerli like rinkeby is so because of that I can remove this logic from here. 

Basically, with original dockerfile there is an option only to run networks that are supported by geth cli.

Those are:
* mainnet (by default)
* --testnet (Ropsten)
* --rinkeby 
* --goerli 

If we need just these networks there is no need for my current change, but if we also want to support others then change that was implemented needs to be used.

**Comment by edisinovcic** [comment]
Won't be needed in current phase.

**Comment by roninkaizen** [comment]
grafik

**Comment by roninkaizen** [comment]
grafik

**Comment by roninkaizen** [comment]
yes, i work with portainer,
sometimes little complicated, but on 4th try,
configured a working config, visible in the screener :)

**Comment by roninkaizen** [comment]
grafik
